include/linux/kfence.h

```diff
@@ -23,7 +23,7 @@ extern char __kfence_pool[KFENCE_POOL_SIZE];
 
 extern struct static_key_false kfence_allocation_key;
 
-static __always_inline bool is_kfence_addr(void *addr)
+static __always_inline bool is_kfence_addr(const void *addr)
```

Comment:
Do we eventually have to add volatile as well? Because we will get the inevitable warnings if somebody passes a volatile pointer.

Reply:
Hmm, k{free,size}() only take 'const void *'. So maybe we can ignore it for now.